### PR TITLE
Adding Jacoco and sonarqube

### PR DESCRIPTION
**Pull request by abhijith-prabhakar**

### CODE REVIEW

File: docs/RELEASE_NOTES.md [hunk 1]
```diff
@@ -1,6 +1,17 @@
 
 # Release notes
```
Comment (fabiocarvalho777) [inline docs/RELEASE_NOTES.md:3]: This is already done in develop branch. We never push directly to master, only to develop.

File: docs/END_USER_DOCUMENT.md [hunk 2]
```diff
@@ -3,7 +3,7 @@
 <div style="text-align:center"><img src="img/logo/mocca_logo_horizontal_lightbluebackground.png" style="background-color:#27a1f2; padding:40px;"/></div>
 <br>
 
-# Mocca end user documentation (version 0.0.3)
+# Mocca end user documentation (version 0.0.4)
```
Comment (fabiocarvalho777) [inline docs/END_USER_DOCUMENT.md:6]: All these changes are done in develop branch already.

File: build.gradle [hunk 3]
```diff
@@ -9,12 +12,13 @@ Set testModules = [
 subprojects {
 
     group = 'com.paypal.mocca'
-    version = '0.0.3'
+    version = '0.0.4-SNAPSHOT'
```
Comment (fabiocarvalho777) [inline build.gradle:15]: This is NOT supposed to be done in master branch, it is done in develop branch (it is done already).

File: .travis.yml [hunk 4]
```diff
@@ -7,6 +7,12 @@ cache:
   - $HOME/.m2
   - $HOME/.gradle
 
+addons:
+  sonarcloud:
+    organization: "paypal"
+    token:
+      secure: "J8c8dsLcZYO0+8Qm5RNPzEyZCYXSbPDey8IwuMm64aGNnJoIwBAmyAPemd5UefF905lOMgoJ8VFzomIKOap+py5b2WUQ0/+9pPESMmvKkpoq+T/kww1ujID43RUJo8+6wrZBGbd0J8zGBk3NKEVozUv/lc27F54OqhgLxstsTLhZbn0eJAOcFZFAiQ7IhcGBnrz70tn4jD7O5O1zKnJT2KBLhhDY+5Xp1RxbZUgvImuPBSKeJSQ1kp7feXwBnYQO3OBj12RrhZUMV1GIAh/BU0KmAFfkmFcEjx2TqHSga/CHT2HrX+3gX4VIsbLfvJ1Jq1GIQ7qk0KDehfIhEf1HAw1KRsnE+z31FbLI8L8u0P2A/D+Y/8v6Qq8rplfLmfKrDudxnLYtRqpJSag5dYBcsAJSzitqjozDWlPwz457pXm4GzbTJ+9N4XFAK9Iqc/TUg+kAAFChixlDiy0rMXPCgVmFX6J1g7sPoKG5xzm0smMUZOGfQhpuC0NxTD/oheyJoHbf8wJZ4QtQq3rvW/mRmQN7IP0gfTXJfPhUe7bkq8Qj3pxDB8YXvRQYU7XGZNsRpGqC/37R7F9naosxr/5pM4qIneSkwrG0bPCCbw5TdDXcjN8X1P/GavpKHil2jytFevkN9QH4QsYJ4Xj/QdwDdYAxV5oP3kqa3n78UPWTyWQ="
```
Comment (fabiocarvalho777) [inline .travis.yml:14]: Secrets must NOT be in our GtHub repo. Please discard this token. If you need a secret used in our CIs, then it needs to be encrypted and uploaded to Travis CI using Travis CLI tool.

**Review by fabiocarvalho777 — CHANGES_REQUESTED** [review]

File: .travis.yml (hunk 4 above)
Comment (abhijith-prabhakar) [reply .travis.yml:14]: this is encrypted token using Travis CLI tool

File: build.gradle (hunk 3 above)
Comment (abhijith-prabhakar) [reply build.gradle:15]: I again issued PR to master branch. Will close and open it on develop

**Pull request closed without merging**